test-suite/consensus-tests/src/lib.rs

```diff
@@ -548,33 +548,35 @@ impl Sandbox {
         }
     }
 
-    pub fn broadcast<T: TryFrom<SignedMessage>>(&self, msg: &Verified<T>) {
+    pub fn broadcast<T>(&self, msg: &Verified<T>)
+    where
+        T: TryFrom<SignedMessage> + std::fmt::Debug,
```

Comment:
Nit / style: Not a fan of fully qualified names. Since `Debug` is used multiple times in the module, it could make sense to import it.